db/migrate/20150402153201_create_votes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateVotes < ActiveRecord::Migration
+  def change
+    create_table :votes do |t|
+      t.boolean :upvote
```

Comment:
booleans should always have a default. you can't really have a boolean field that is true, false, or nil. By its binary nature, it is either true or false. I would set a reasonable default here, as well as a null: false contraint